atk-sys/src/lib.rs

```diff
@@ -1056,7 +1056,6 @@ pub struct AtkTableCellIface {
 impl ::std::fmt::Debug for AtkTableCellIface {
     fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
         f.debug_struct(&format!("AtkTableCellIface @ {:?}", self as *const _))
-            .field("parent", &self.parent)
```

Comment:
Why are these all removed?

Reply:
I have literally no idea... I guess it comes from [there](https://github.com/gtk-rs/gir/pull/843)?

cc @EPashkin

Reply:
Seems this because field is not readable and private
```diff
-    <record name="TableCellIface" c:type="AtkTableCellIface" glib:is-gtype-struct-for="TableCell">
-      <field name="parent">
+    <record name="TableCellIface" c:type="AtkTableCellIface" glib:is-gtype-struct-for="TableCell" version="2.12">
+      <field name="parent" readable="0" private="1">
```

Reply:
Not sure why but we don't "debug" private fields:
https://github.com/gtk-rs/gir/blob/master/src/codegen/sys/fields.rs#L127